source/common/http/conn_manager_impl.cc

```diff
@@ -64,6 +64,10 @@ ConnectionManagerImpl::ConnectionManagerImpl(ConnectionManagerConfig& config,
       runtime_(runtime), local_info_(local_info), cluster_manager_(cluster_manager),
       listener_stats_(config_.listenerStats()) {}
 
+const std::unique_ptr<const Http::HeaderMap> ConnectionManagerImpl::CONTINUE_HEADER{
```

Comment:
Technically I think this breaks the non-POD rule (though I don't care). Might want to expose via private function and use CONSTRUCT_ON_FIRST_USE.

Reply:
Yeah, I was mentally claiming it was moving an existing non-POD header map but I should really just clean as I move it.